_posts/2018-11-01-paste-from-register-in-command.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: paste from register in command
+date: 2018 Nov 01 07:09:25
```

Comment:
Does this need to be YYYY-MM-DD HH:MM:SS or does this work?

Reply:
Works in preview mode.